README.md

```diff
@@ -154,6 +154,60 @@ yarn run dev
 ```
 If changes aren't detected, you might need to increase `fs.inotify.max_user_watches`. See <https://webpack.js.org/configuration/watch/#not-enough-watchers>.
 
+#### Deploying a Custom Image to an OpenShift Cluster
+
+Once you have made changes locally, these instructions will allow you to push changes to an
+ OpenShift cluster for others to review.  This involves building a local image, pushing the
+ image to an image registry, then updating the OpenShift cluster to pull the new image.
+
+##### Prerequisites
+
+1. Docker v17.05 or higher
+2. An image registry like [quay.io](https://quay.io/signin/) or [Docker Hub](https://hub.docker.com/)
+
+##### Steps
+1. Create a repository in the image registry of your choice to hold the image
+2. Build Image `docker build -t <your-image-name> [<your-git-url> | <path-to-dockerfile>]`
+ 
+    Build from Git Repository:
+    ```
+    docker build -t quay.io/myaccount/console:latest github.com/myaccount/console#latestchanges
+    ```
+    ...or from console directory:
+    ```
+    docker build -t quay.io/myaccount/console:latest .
+    ```
+3. Push Image to Image Registry `docker push <your-image-name>`.  Make sure docker is logged into your image registry!  Here we are pushing to a repository named 'console' on quay.io:
+    ```
+    docker push quay.io/myaccount/console:latest
+    ```
+4. Put the console operator in unmanaged state.  Make sure you `oc login` to remote OpenShift cluster. 
+    ```
+    oc edit consoles.operator.openshift.io cluster
```

Comment:
This can be a one-liner:

```
oc patch consoles.operator.openshift.io cluster --patch '{ "spec": { "managementState": "Unmanaged" } }' --type=merge
```

Reply:
fixed